cmd/dicomutil/dicomutil.go

```diff
@@ -62,7 +62,7 @@ func main() {
 		// Begin parsing
 		parsedData, err = p.Parse(dicom.ParseOptions{})
 		if err != nil {
-			log.Panic("error parsing", err)
+			log.Panic("error parsing: ", err)
```

Comment:
nit: since you're updating this anyway, can probably do something like `log.Panicf("error parsing: %v", err")`